src/ts/core/roam/roam.ts

```diff
@@ -144,6 +156,10 @@ export const Roam = {
         await Mouse.hover(foldButton)
         await Mouse.leftClick(foldButton)
     },
+
+    async followLinkUnderCursor(shiftKey: boolean = true) {
```

Comment:
not sure if this fits the Roam object 🤔

Reply:
I'll keep it local to `random-page` for now